src/lighteval/main_accelerate.py

```diff
@@ -44,7 +44,7 @@ def accelerate(  # noqa C901
     model_args: Annotated[
         str,
         Argument(
-            help="Model arguments in the form key1=value1,key2=value2,... or path to yaml config file (see examples/model_configs/base_model.yaml)"
+            help="Model arguments in the form key1=value1,key2=value2,... or path to yaml config file (see examples/model_configs/transformers_model.yaml)"
```

Comment:
Need to rename the file
- examples/model_configs/base_model.yaml
to
- examples/model_configs/transformers_model.yaml